clang/include/clang/Sema/Sema.h

```diff
@@ -9739,6 +9739,9 @@ class Sema final : public SemaBase {
                      const PartialDiagnostic &CandidateDiag,
                      bool Complain = true, QualType TargetType = QualType());
 
+  FunctionDecl *getMoreConstrainedFunction(FunctionDecl *FD1,
```

Comment:
You put this declaration into `SemaTemplateDeclaration` section of `Sema.h`, but the function is implemented in `SemaTemplate.cpp`. They should be in sync.